plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImpl.kt

```diff
@@ -21,8 +21,12 @@ import kotlin.properties.Delegates
  * It is responsible for:
  *   - Storing all the [ValueAnimator]'s that could be run. Only specific camera animators could be used in this plugin.
  *   - Controlling animation execution - only one [ValueAnimator] of certain type could be run at a time.
- *   If some another animation with same [CameraAnimator.type] is about to start previous one will be cancelled.
+ *     If some another animation with same [CameraAnimator.type] is about to start previous one will be cancelled.
  *   - Giving possibility to listen to [CameraOptions] values changes during animations via listeners.
+ *   - If several animations of different [CameraAnimator.type] are running simultaneously map camera
+ *     will be updated only on oldest animation update (the oldest is the one that was started first).
+ *     That actually means that animation start order matters. High-level animations [flyTo] and [easeTo]
+ *     will always trigger camera center animator first.
```

Comment:
Would be great to also explain why (grouping updates to save some CPU time).

BTW, is this actually still necessary if the Choreographer is now coordinating when a frame is rendered? In the past, each change to GL-Native (like `setCamera`) would request a new frame, but that isn't the case anymore, is it?
Is this grouping of updates still necessary in this case to save a little bit of C++ code from running, if those updates do not trigger a frame re-render anyway? Did you have a chance to measure the performance difference versus calling `setCamera` on each animation tick?

Reply:
@LukasPaczos we need to take into account that we reduce JNI calls in any case now + `setCamera` impl in C++ is also not that trivial so this optimisation __should__ be a bit valuable. 
I really liked your idea to add some kind of flag to turn this optimisation on / off. I'll create a follow-up task and measuring could organically happen there. WDYT?

Reply:
https://github.com/mapbox/mapbox-maps-android/issues/366

Reply:
Sound good :+1: